front/cypress/e2e/admin/pages_and_menu/update_homepage_hero_banner_content.cy.ts

```diff
@@ -77,6 +77,31 @@ describe('Admin: update Hero Banner content', () => {
     );
   });
 
+  it('uploads and crops banner image', () => {
+    cy.intercept('PATCH', '**/home_page').as('saveHomePage');
+    cy.setAdminLoginCookie();
+    cy.visit('admin/pages-menu/homepage/homepage-banner/');
+    cy.acceptCookies();
+
+    cy.get('[data-cy="e2e-fixed-ratio-layout-option"]').click();
+    cy.get('[data-cy="e2e-remove-image-button"]').click();
+    cy.get('[data-cy="e2e-homepage-banner-image-dropzone"] input').attachFile(
+      'testimage.png'
+    );
+
+    cy.get('.e2e-submit-wrapper-button').click();
+
+    cy.wait('@saveHomePage').then((interception) => {
+      const img = new Image();
+      img.src = interception.request.body.home_page.header_bg;
+      img.decode().then(() => {
+        expect(img.width / img.height).to.eq(3);
+      });
+    });
+
+    cy.get('.e2e-submit-wrapper-button').contains('Success');
+  });
```

Comment:
As Iva said, we want to test the whole user flow. So what's still missing here is navigating to the homepage and checking the fixed-ratio banner layout is present.